starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -827,6 +846,11 @@ public void release() {
       Log.e(TAG, "Cannot release media codec", e);
     }
     mMediaCodec = null;
+    if (mCallbackThread != null) {
```

Comment:
Consider to add `synchronized (this) {}` block as the callbacks are guarded by `this` object.

Reply:
Revised